sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UnusedPrivateMember.cs

```diff
@@ -454,22 +450,46 @@ private void ConditionalStore<TSymbol>(TSymbol symbol, Func<TSymbol, bool> condi
             }
 
             private ISymbol GetDeclaredSymbol(SyntaxNode syntaxNode) =>
-                this.getSemanticModel(syntaxNode).GetDeclaredSymbol(syntaxNode);
+                getSemanticModel(syntaxNode).GetDeclaredSymbol(syntaxNode);
 
-            private bool IsDeclaredInPartialClass(IMethodSymbol methodSymbol)
+            private void StoreRemovableVariableDeclarations(BaseFieldDeclarationSyntax node)
+            {
+                foreach (var variable in node.Declaration.Variables)
+                {
+                    var symbol = GetDeclaredSymbol(variable);
+                    if (IsRemovableMember(symbol))
+                    {
+                        PrivateSymbols.Add(symbol);
+                        FieldLikeSymbols.Add(symbol, variable);
+                    }
+                }
+            }
+
+            private static bool IsEmptyConstructor(BaseMethodDeclarationSyntax constructorDeclaration) =>
+                !constructorDeclaration.HasBodyOrExpressionBody() ||
+                (constructorDeclaration.Body != null && constructorDeclaration.Body.Statements.Count == 0);
+
+            private static bool IsDeclaredInPartialClass(ISymbol methodSymbol)
             {
                 return methodSymbol.DeclaringSyntaxReferences
                     .Select(GetContainingTypeDeclaration)
                     .Any(IsPartial);
 
-                TypeDeclarationSyntax GetContainingTypeDeclaration(SyntaxReference syntaxReference) =>
+                static TypeDeclarationSyntax GetContainingTypeDeclaration(SyntaxReference syntaxReference) =>
                     syntaxReference.GetSyntax().FirstAncestorOrSelf<TypeDeclarationSyntax>();
 
-                bool IsPartial(TypeDeclarationSyntax typeDeclaration) =>
+                static bool IsPartial(TypeDeclarationSyntax typeDeclaration) =>
                     typeDeclaration.Modifiers.AnyOfKind(SyntaxKind.PartialKeyword);
             }
 
-            private bool IsRemovable(ISymbol symbol) =>
+            private static bool IsRemovableMethod(IMethodSymbol methodSymbol) =>
+                IsRemovableMember(methodSymbol) &&
```

Comment:
Can you move the `&&` at the beginning of next line? In the whole file.

Reply:
Can you do this in the whole file? I mentioned it in the comment to avoid creating duplicate threads.

Reply:
I would like to focus this cleanup on static methods and generic types. Baby steps 😄

Reply:
I would like to clean as you code :) There are not that many cases and it will help every other PR to be easier to read.

Reply:
This is clean as you code made in baby steps 😄

Reply:
But sure, I can make on more PR to handle other topics as well.